protos/flyteidl/event/event.proto

```diff
@@ -167,4 +197,14 @@ message TaskExecutionMetadata {
         INTERRUPTIBLE = 1;
     }
     InstanceClass instance_class = 16;
+
+    // Contains various identifiers for resources used during execution.
```

Comment:
nit: can we order them by the field id? sorry!

Reply:
@kumare3 done